src/controllers/country.controller.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { Response, Request } from 'express';
+import { getCountry } from '../services/country.service';
+import { getPhoneNumberByCountry } from '../services/phoneNumber.service';
+import logger from '../config/logger';
+import { REGEX_COUNTRY_CODE } from '../utils/constants';
+
+export const getPhoneNumbers = async(req: Request, res: Response) => {
+  try {
+    const countryCode = req.params.code;
+    
+    if (!validateCountryCode(countryCode)) {
+      res.render('404');
+    } else {
+      const country = await getCountry(countryCode);
+
+      if (!country) {
```

Comment:
Ý anh Phương là thế này để hạn chế block if else c
```suggestion
      if (!country) return res.render('404');
      
      const phoneNumbers = await getPhoneNumberByCountry(country._id);
      const result = {country, phoneNumbers}
       res.render('countries/show', result);
```